chapter2/tests/t19/terraform/_modules/container/delete_folder.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Script for deleting a folder with a generated HTML file during the execution of the destroy operation.
+
+target_folder="${PWD}/../../../"
+folders=("qa" "dev" "prod" "stage")
```

Comment:
not a good solution - as you delete ALL. You should only delete one per env.

Reply:
> not a good solution - as you delete ALL. You should only delete one per env.


@dmitry-mightydevops 
Done